src/main/webapp/timeline.js

```diff
@@ -2,4 +2,55 @@ function addNode() {
     var timeline = document.getElementById("timeline");
     console.log(timeline.innerHTML)
     timeline.innerHTML = timeline.innerHTML.concat("<span class=\"timeline-node\"><>--</span>");
-}
\ No newline at end of file
+}
+
+function loadResults() {
+  fetch('/timeline_data').then(response => response.json()).then((tricks) => {
+    const trickNode = document.getElementById('trick-results');
```

Comment:
Instead of trick-results, this should be referencing "timeline"

Reply:
Thanks!